test/e2e_node/garbage_collector_test.go

```diff
@@ -231,7 +231,7 @@ func containerGCTest(f *framework.Framework, test testRun) {
 		AfterEach(func() {
 			for _, pod := range test.testPods {
 				By(fmt.Sprintf("Deleting Pod %v", pod.podName))
-				f.PodClient().DeleteSync(pod.podName, &metav1.DeleteOptions{}, defaultRuntimeRequestTimeoutDuration)
+				f.PodClient().DeleteSync(pod.podName, &metav1.DeleteOptions{}, podDisappearTimeout)
```

Comment:
Might be worth just doing a quick pass over the e2e node suite and rename constants and move them to a single file